tests/UnitTests/Tests.fs

```diff
@@ -208,4 +208,118 @@ let ``test print for issue 14``() =
         with
             | e -> false
 
-    Assert.True(print)
\ No newline at end of file
+    Assert.True(print)
+
+
+// DynamicObj of json tests
+
+let private minifyJson(json:string) = 
+    json
+        .Replace(" ","")
+        // if i add this line, my tests break
```

Comment:
You might want to implement the tests using the JSON as embedded resources to prevent newline problems. see https://github.com/fslaborg/FSharp.Stats/commit/23b49a95a8167829643cf743d296a14b8081907c